insights/client/archive.py

```diff
@@ -22,45 +22,75 @@ class InsightsArchive(object):
     """
     This class is an interface for adding command output
     and files to the insights archive
+
+    Attributes:
+        config          - an InsightsConfig object
+        tmp_dir         - a temporary directory in /var/tmp
+        archive_dir     - location to collect archive data inside tmp_dir
+        archive_tmp_dir - a temporary directory to write the final archive file
+        archive_name    - filename of the archive and archive_dir
+        cmd_dir         - insights_commands directory inside archive_dir
+        compressor      - tar compression flag to use
+        tar_file        - path of the final archive file
     """
     def __init__(self, config):
         """
         Initialize the Insights Archive
-        Create temp dir, archive dir, and command dir
         """
         self.config = config
+        # input this to core collector as `tmp_path`
         self.tmp_dir = tempfile.mkdtemp(prefix='/var/tmp/')
+
+        # we don't really need this anymore...
```

Comment:
I'd like to take it out as a future PR just to avoid rocking the boat too much at once. It's a holdover from the "uberarchive" stuff in 2016 and there may be QE tests built around it by now